Tests/ProjLintKitTests/Rules/FileExistenceRuleTests.swift

```diff
@@ -39,4 +39,72 @@ final class FileExistenceRuleTests: XCTestCase {
             XCTAssert(violations.isEmpty)
         }
     }
+
+    func testAllowedPathsWithValidRegex() {
+        resourcesLoaded([infoPlistResource]) {
+            let optionsDict = ["allowed_paths_regex": ["Sources/SuportingFiles/Info\\.plist"]]
+            let rule = FileExistenceRule(optionsDict)
+
+            let violations = rule.violations(in: Resource.baseUrl)
+            XCTAssertEqual(violations.count, 0)
+        }
+
+        resourcesLoaded([infoPlistResource]) {
+            let optionsDict = ["allowed_paths_regex": ["Sources/SuportingFiles/Info2\\.plist"]]
```

Comment:
Please use [raw strings](https://www.hackingwithswift.com/articles/162/how-to-use-raw-strings-in-swift) for regex patterns. You could even do that if the String didn't contain any escaped characters, but that's up to you.

Reply:
Done